src/EcommerceStatsBundle/README.md

```diff
@@ -0,0 +1,10 @@
+# eCommerce Statistics Bundle
+
+This bundle generates statistics reports and exposes them in multiple ways
+in the backend.
```

Comment:
Please add an example how the bundle can be customized to add new columns, different statistics, or additional filter.

Reply:
:heavy_check_mark: Added examples to the `README.md` file